torch/optim/radam.py

```diff
@@ -313,7 +313,9 @@ def _compute_adaptive_lr():
             return (bias_correction2**0.5) / exp_avg_sq_sqrt
```

Comment:
not related to this PR, but maybe `_dispatch_sqrt` (in this file and elsewhere) can be replaced by simply `** 0.5` which is supposed to work both for python scalars and for tensors (this is already used in most occurences in this file as well, the only dispatch_sqrt usage remaining)